requirements.txt

```diff
@@ -12,3 +12,4 @@ progressbar2
 nltk
 numpy
 requests
+psycopg2==2.6.1
```

Comment:
Don't hardcode a version number for dependencies. Our approach is to set a minimum version number and revise the code if a later version breaks it.